config/routes.rb

```diff
@@ -1,4 +1,10 @@
 Rails.application.routes.draw do
+  get 'welcome/index'
```

Comment:
Line 2 can be safely removed as the root `/` path is showing the `welcome/index` page.